### PR TITLE
SDK-109 Introduce wizard for modules

### PR DESCRIPTION
**Pull request by PawelGutkowski**
This PR introduces changes:
- replace invoking `openmrs-contrib-maven-plugin-module-wizard` with own wizard and execute project generation without `MojoExecutor`
- change refapp module archetype parameters to align to convention from other archetypes (`A-moduleName` to `module-name` etc.) 
- add tests to CreateProject and delete redundant properties from test pom.xml

### CODE REVIEW

File: integration-tests/src/test/java/org/openmrs/maven/plugins/CreateProjectIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    @After
+    public void resetPom() throws Exception {
+        saveTestPom(testDirectory, model);
+    }
+
+    @Test
+    public void createProject_shouldCreateRefappModuleProject() throws Exception{
+        addTaskParam("moduleType", "refapp");
```
Comment (rkorytkowski) [inline integration-tests/src/test/java/org/openmrs/maven/plugins/CreateProjectIntegrationTest.java]: Could we rename it to just type? I envision create-project being called for OWAs, creating distributions, etc. We'll have the following types to start with: platform-module, referenceapplication-module

Comment (PawelGutkowski) [inline integration-tests/src/test/java/org/openmrs/maven/plugins/CreateProjectIntegrationTest.java]: @rkorytkowski done!

**Comment by dkayiwa** [comment]
@rkorytkowski and @PawelGutkowski did you see this? https://ci.openmrs.org/browse/SDK-SDK-66

**Comment by PawelGutkowski** [comment]
@dkayiwa we didn't, thanks for noticing, we are working on it

**Comment by dkayiwa** [comment]
@rkorytkowski and @PawelGutkowski any updates on this? Or should i disable the CI plan for now? It has been failing for a number of days. :)

**Comment by PawelGutkowski** [comment]
@dkayiwa problem is quite malicious, error is not occuring on our machines and I didn't manage to reproduce it yet, but I am working on it. Please don't disable CI plan, without it we can't see if we fixed the problem.

**Comment by dkayiwa** [comment]
@PawelGutkowski which operating system are you using?

**Comment by PawelGutkowski** [comment]
@dkayiwa Ubuntu 14.04, but I don't think it has to do anything with it. We found source of error, I've opened PR, after merge CI should pass.

**Pull request merged**